src/optimizer.cls.php

```diff
@@ -43,6 +43,11 @@ public function html_min($content, $force_inline_minify = false)
 		if ($force_inline_minify) {
 			$options['jsMinifier'] = __CLASS__ . '::minify_js';
 		}
+		
+		$skip_comments = $this->conf(Base::O_OPTM_HTML_SKIP_COMMENTS);
+		if( !empty($skip_comments) ){
+			$options['skipComments'] = $skip_comments;
```

Comment:
Glad you update here correspondingly w/ default type changes above. Well done!

Reply:
Thank you! :)